source/install/deprecated-features.rst

```diff
@@ -6,6 +6,14 @@ This page describes features that are removed from support for Mattermost, or wi
 Removed features in upcoming versions
 -------------------------------------
 
+Mattermost Server v9.0.0
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Boards is moving to a community plugin.
+- Removing support for some prepackaged plugins.
+- Removing deprecated Insights feature.
+- In-product GIF picker powered by Gfycat `is being discontinued <https://techcrunch.com/2023/07/05/gfycat-shuts-down-on-september-1/>`_. Gfycat will be replaced with GIPHY in the v9.0 release.
```

Comment:
Replace with:
Gfycat as a GIF service is being [discontinued](https://techcrunch.com/2023/07/05/gfycat-shuts-down-on-september-1/) on September 1, 2023. The in-product GIF picker currently powered by Gfycat will be replaced with GIPHY in the 9.0 release.

Reply:
@amyblais do we typically mention the fact that we'll also be releasing dot releases and in ESRs?

Reply:
I added "...in the 9.0 release and in our supported versions".

Reply:
Great, thanks Amy!